nyaa/views/torrents.py

```diff
@@ -40,7 +40,16 @@ def view_torrent(torrent_id):
 
         if comment_form.validate():
             comment_text = (comment_form.comment.data or '').strip()
-
+            mentioned = [s for s in comment_text.split() if '@' in s]
+            if len(mentioned) > 0:
```

Comment:
1. This won't work for users that have a space in their username (I think I saw at least some that does).
But that's not your issue. Your code works for all other cases (which is most of the users).

2. This is more Pythonic: `if mentioned:`
This means, if `mentioned` **isn't** Falsey (`False`, `None`, `empty string`, `empty list`, etc...)